neon/datasets/tests/test_cifar100.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import shutil
+from nose.plugins.attrib import attr
+
+from neon.datasets.cifar100 import CIFAR100
+from neon.backends.cpu import CPU
+from neon.backends.par import NoPar
+
+
+class TestCIFAR100(object):
+
+    tmp_repo = os.path.join(os.path.dirname(__file__), 'repo')
+
+    def setup(self):
+        os.makedirs(self.tmp_repo)
+
+    def teardown(self):
+        shutil.rmtree(self.tmp_repo, ignore_errors=True)
+
+    @attr('slow')
+    def test_fine_labels(self):
+        data = CIFAR100(coarse=False, repo_path=self.tmp_repo)
+        data.backend = CPU(rng_seed=0)
+        data.backend.actual_batch_size = 128
+        data.backend.par = NoPar(data.backend)
```

Comment:
Sorry, one final thing here.  The `data.backend.par = NoPar(data.backend)` line currently won't work (and is in fact broken in datasets/tests/test_mnist.py too!).  You'll need to replace that line with:

    par = NoPar()
    par.associate(data.backend)

That should get the tests passing